salt/modules/kerberos.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+'''
+Manage Kerberos KDC
+
+:configuration: In order to manage your KDC you will need to generate a keytab
+that can authenticate without requireing a password.
+
+    .. code-block:: bash
+        # ktadd -k /root/secure.keytab kadmin/admin kadmin/changepw
+
+On the KDC minion you will need to add the following to the minion
+configuration file so Salt knows what keytab to use and what principal to
+authenticate as.
+
+    .. code-block:: yaml
+        auth_keytab: /root/auth.keytab
+        auth_principal: kadmin/admin
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import Salt libs
+import salt.utils
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    if salt.utils.which('kadmin'):
+        return True
+
+    return False
+
+
+def __execute_kadmin(cmd):
+    '''
+    Execute kadmin commands
+    '''
+    ret = {}
+
+    auth_keytab = __opts__.get('auth_keytab', None)
```

Comment:
We should be able to use config.get for these values, and they should likely be renamed to something like
krb:auth_keytab